### DISCUSSION TITLE
DNS resolve between Namecheap (domain) and Vercel (hosting) is not working

### DISCUSSION THREAD

**Discussion by LFBFerreira** [vercel/community#6713] [Account Issue] [answered]
### Question

Hello, i am quite stuck and have tried many things but i still can't get this to work. Here is a summary of my problem:
 
A couple of weeks ago when visiting my website, luisferreira.tech, there was an SSL error due to a problem with a certificate. I contacted support at Namecheap and they quickly fixed that error, but since then my domain never pointed correctly at my hosting service anymore. 
I have a domain on NameCheap and my website is hosted on Vercel.com (portfolio-website-three-ashy.vercel.app). For this connection to work Vercel asks me to add custom nameservers and a DNS A Record to NameCheap. 
I added ns1.vercel-dns.com and ns2.vercel-dns.com as custom Nameservers, and a A record pointing to @ (luisferreira.tech) and IP 76.76.21.21.
I did this already a few days ago and my website is still down, if i access it with a browser i get the error DNS_PROBE_FINISHED_NXDOMAIN, and if i use https://dnschecker.org/ to find A records for luisferreira.tech nothing is found.

The support center from NameCheap replied with the following: 

"Per our check, your domain name luisferreira.tech is pointed to ns2.vercel-dns.com. and ns1.vercel-dns.com. servers. Unfortunately, as we can see there is no A record created for your domain on those nameservers.
It is not technically possible to manage the host records in your Namecheap account when you point your domain name to the nameservers provided by a third-party company.
This is because zone file that includes host records for your domain name is stored on the servers of your hosting company so the domain is manageable only from their end.
So feel free to contact Versel to assist you with A record setup."

If Vercel shows me on it's Domain's user interface that the domain is correct, luisferreira.tech (does it matter if there's a dot at the end or not?) and the A record should point to that IP, why do they say there is no A record for my site on Vercel's DNS servers? Should i use NameCheap's basic DNS server or do i really need to use Vercel's nameservers?

I would appreciate some help with this.

**Comment by LFBFerreira** [comment, marked as answer]
I tried reseting the Nameservers and instead of using Vercel's i just used the BasicDNS from Namecheap, but stil with the A record. It's working!

**Reply by shakedty** [reply]
can you explain how you did it exactly?  ihave excatly the same issue
it solves domain on windows but gets bad ssl on mac.

[a comment by an automated account is not shown]
